spring-cloud-netflix-zuul/src/main/java/org/springframework/cloud/netflix/zuul/filters/SimpleRouteLocator.java

```diff
@@ -16,26 +16,38 @@
 
 package org.springframework.cloud.netflix.zuul.filters;
 
+import static java.lang.String.format;
+import static java.util.function.Function.identity;
+import static java.util.stream.Collectors.toMap;
+import static java.util.stream.Collectors.toSet;
+
 import java.util.ArrayList;
 import java.util.Collection;
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.Set;
 import java.util.concurrent.atomic.AtomicReference;
+
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.springframework.cloud.netflix.zuul.filters.ZuulProperties.ZuulRoute;
 import org.springframework.cloud.netflix.zuul.util.RequestUtils;
 import org.springframework.core.Ordered;
+import org.springframework.http.HttpMethod;
 import org.springframework.util.AntPathMatcher;
+import org.springframework.util.CollectionUtils;
 import org.springframework.util.PathMatcher;
 import org.springframework.util.StringUtils;
 
+import com.google.common.collect.Sets;
```

Comment:
Please remove `com.google` references

Reply:
Okay, didn't know that's forbidden to use. Does it make sense to remove Guava from the dependencies?

Reply:
That is what I'm asking

Reply:
Maybe I wasn't clear enough with 'dependencies'. I meant the Maven dependencies. Of course I'll remove it from the imports but maybe it makes sense to remove the Maven dep as well.

Reply:
Yes

Reply:
Just checked it and it seems that the Guava lib is coming through this chain:
    [INFO] +- com.netflix.zuul:zuul-core:jar:1.3.1:compile
    [INFO]  |  +- com.netflix.archaius:archaius-core:jar:0.7.5:compile
    [INFO]  |   |  +- com.google.guava:guava:jar:18.0:compile

So I'll leave it as it is and remove the import only.